modules/vaos/app/controllers/vaos/v0/request_eligibility_criteria_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module VAOS
+  module V0
+    class RequestEligibilityCriteriaController < VAOS::V0::BaseController
+      def index
+        response = systems_service.get_request_eligibility_criteria(get_params[:site_codes])
+        render json: VAOS::V0::RequestEligibilityCriteriaSerializer.new(response)
+      end
+
+      private
+
+      def get_params
```

Comment:
nitpick, this method could be more descriptive to what it's actually doing but total nitpick